LayoutTests/fast/forms/auto-fill-button/hide-auto-fill-button-when-input-becomes-readonly.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <body>
 <p>This tests that the AutoFill button does not render when the field becomes read only.</p>
-<input type="password" id="password" style="width: 100px; margin: 0;">
+<input type="password" id="password" style="width: 100px; margin: 5px;">
```

Comment:
Why is the additional margin necessary?

Reply:
I do not know and actually cannot reproduce the issue locally, which makes it hard to investigate. 

What I do know is that the patch that caused the regression removed the intrinsic margins for form controls. As a result, I tried to added margins back to see if it fixed the regression and it seems it did according to the bots.

Also note that this test is about checking that some auto fill button does not show, it has nothing to do with margins per se.